src/schema.ts

```diff
@@ -9,13 +9,16 @@ export const urlSchema = z
   .string()
   .url()
   .startsWith(urlStart, `URL must start with [${urlStart}].`)
-  //.co will cause fetch to time out
-  .endsWith(urlEnd, `URL must end with [${urlEnd}].`)
   .min(
     urlStart.length + urlEnd.length + 1,
     'Domain name must be at least 1 character long.'
   )
   .refine(
     (url) => !url.endsWith('-admin.okta.com'),
     'Organisation URL should not be the admin URL. Please remove "-admin" and try again.'
+  )
+  .refine(
+    //.co will cause fetch to time out
+    (url) => url.endsWith(urlEnd) || url.endsWith(`${urlEnd}/`),
```

Comment:
**Question:** does javascript have a structured URL class that allows you to extract the hostname? That would make this match simpler.

Reply:
I've used readonlyURL from our readonly-types library
![image](https://github.com/agiledigital-labs/oktagon/assets/124639768/498195de-7554-4924-a95f-bd7d77cc7a31)

Reply:
modified in commit: ea34b28daf396fa892cb99197bb22682e159febf